examples/theming/App.svelte

```diff
@@ -1,25 +1,39 @@
 <script>
     import ExtendedTable from "../../src/ExtendedTable.svelte";
-    import {default as data} from '../../fixtures/fixtures.json';
+    import {default as rawData} from '../../fixtures/fixtures.json';
 
     let columnDefinition = [
         {
             title: 'Username',
             propertyPath: 'username',
+            headerClassName: 'headline__username',
         },
         {
             title: 'Name',
             value: (rowData) => `${rowData.title} ${rowData.first_name} ${rowData.last_name}`,
             propertyPath: 'last_name',
+            className: {
+                propertyPath: 'username',
+                value: (rowData, column, columnIndex, rowIndex) => rowData.username === 'goldenkoala410' ? 'success' : '',
```

Comment:
`column` is the current column spec defined, how is it useful in the function? Even if it is useful, it's accessible here as the user is defining it. Remove? Remove `row` in row config function too?

Reply:
Good catch! That parameter is indeed not very useful. I removed it.